src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -98,12 +97,12 @@ void MulticopterLandDetector::_update_topics()
 		_flag_control_climb_rate_enabled = vehicle_control_mode.flag_control_climb_rate_enabled;
 	}
 
-	if (_params.useHoverThrustEstimate) {
+	if (_hover_thrust_estimate_sub.updated()) {
 		hover_thrust_estimate_s hte;
 
 		if (_hover_thrust_estimate_sub.update(&hte)) {
```

Comment:
```suggestion
		if (_hover_thrust_estimate_sub.copy(&hte)) {
```

Since you already have the `updated` above.